### DISCUSSION TITLE
建议开放特殊SSH端口

### DISCUSSION THREAD

**Discussion by juicyroastgoose** [midoks/mdserver-web#408] [Ideas]
面板能识别当前的非22的SSH端口，但是不会自动开放。我昨天安装了面板之后发现我特殊端口的SSH连不上了，以为是小鸡挂了。还好面板还能进去，手动开放了SSH端口。

既然面板能自动识别非标准SSH端口，建议识别后自动开放这个端口~

**Comment by midoks** [comment]
已经解决。
